src/views/RepoDetailIssueView/IssueChart.tsx

```diff
@@ -146,13 +150,21 @@ const IssueChart: React.FC<IssueChartProps> = (props) => {
     const instance = echarts.getInstanceByDom(chartDOM as any);
     if (instance) {
       instance.setOption(option);
+      if (onClick) {
+        instance.on('click', (params) => {
+          onClick(issueclickparams, params);
+        });
+      }
     }
   }, []);
 
   return <div ref={divEL} style={{ width, height }}></div>;
 };
 
+let issueclickparams: { data: number[]; marker: any; seriesName: any };
```

Comment:
Since only `issueclickparams.data[0]` is read as ym, maybe we can use a variable called `curMonth` to store the property and pass the string variable to onClick(curMonth, params) directly.

In my view this is more clear.